features/window-controls-overlay.yml

```diff
@@ -0,0 +1,18 @@
+name: Window Controls Overlay
```

Comment:
I hate it when we have to capitalize features, but this one is so weirdly named I don't think we have a reasonable alternative.